modular/manager/sp_exit_scheduler.go

```diff
@@ -316,6 +309,7 @@ func (s *SPExitScheduler) AddSwapOutToTaskRunner(swapOut *virtualgrouptypes.MsgS
 			log.Errorw("failed to add swap out to task runner", "gvg_unit", gUnit, "error", err)
 			return err
 		}
+		log.Infow("succeed to add gvg unit to task runner", "gvg_unit", gUnit)
```

Comment:
Also need to print the info of sp and gvg.

Reply:
gUnit already contains all.